ispo-working-group-charter.md

```diff
@@ -0,0 +1,39 @@
+The purpose of this working group is to support InnerSource Program Offices (ISPOs) with materials needed to establish themselves, teach and scale InnerSource in their respective company.
+This material could be directed towards senior leaders, managers, or individuals in a variety of roles.
+
+# Summary
+
+We believe that most training and teaching content can be shared across companies when those institutions share the same relevant points of context.
+While not every company is exactly the same, not every company is wholly different, either.
+For each challenge in InnerSource, we believe that there are a small number of ways of common ways to address it that should work for most companies.
+Our goal is to document these sets of solutions to challenges, make it easy for ISPOs to find which apply to them, and then to produce sharable, open source implementations of these solutions.
+These implementations will likely be documents, videos, or code, each of which is generic enough to be used across companies,
+yet specific enough that people in the company can leverage them to solve their problems without needing notable amounts of company-specific work.
+
+# Examples
+
+As an example, to the challenge of `How do I sell InnerSource to Senior Leadership?`,
+this working group could ensure that there are video clips explaining various benefits of InnerSource from a Senior Leader perspective.
+There would be different clips that highlight different benefits of InnerSource,
+and the ISPO could select those videos that highlight the benefits that their Senior Leaders care about.
+
+As another example, to the challenge of `How do I show InnerSource KPIs?`,
+this working group could ensure that there is a software project that will produce charts of InnerSource activity from sources like _GitHub_ and Jira.
+The project could be configured in a (finite) number of ways, depending on varying setup of _GitHub_ and Jira that affects tha report views.
+
+These two examples follow the same pattern - providing directly-usable materials that help ISPOs to advance their mission in their company.
+
+# Implementation
+
+This working group will track the discovery, production, and spread of solutions to challenges with InnerSource.
+We will track this production on a Kanban board with the following columns:
+
+1. **Challenge**.  This means that we've identified such-and-such a thing as a challenge.
+1. **Proposed**.  We have a proposal for a solution to the challenge.
+This could be to write an article, code up a tool, create a training video, etc.
+1. **Internal**.  We have implemented the solution internally in a company.
+1. **External**.  We have made the solution available externally.
+1. **Validated**.  More than one company is using the solution.
```

Comment:
I like this structured approach. The way it is decsribed here, reminds me of InnerSource patterns. Thus, I would expect that in many cases, these challenges together with the corresponding solutions would form (a) pattern(s).

Reply:
Yes!  Suspect so!  A lot of the work here may end up producing patterns.  The working group will help us to organize ourselves sufficiently with the pre-work needed to crank out more patterns.